msg/RadarTrack.msg

```diff
@@ -0,0 +1,23 @@
+# This message relates only to FMCW radar.  
+# All variables below are relative to the radar's frame of reference.
+# This message is not meant to be used alone but as part of a stamped or array message.
+
+# Object classifications (Additional vendor-specific classifications are permitted eg. 3 = Car)
+uint8 NO_CLASSIFICATION=0
+uint8 STATIC=1
+uint8 DYNAMIC=2
+
+
+uuid_msgs/UniqueID uuid                     # A unique ID of the object generated by the radar.
+
+                                            # Note: The z component of these fields is ignored for 2D tracking.
+geometry_msgs/Point position                # x, y, z coordinates of the centroid of the object being tracked.
+geometry_msgs/Vector3 velocity              # The velocity of the object in each spatial dimension.
+geometry_msgs/Vector3 acceleration          # The acceleration of the object in each spatial dimension.
+geometry_msgs/Vector3 size                  # The object size as represented by the radar sensor eg. length, width, height OR the diameter of an ellipsoid in the x, y, z, dimensions
+                                            # and is from the sensor frame's view.
+uint8 classification                        # An optional classification of the object (see above)
+float64[9] position_covariance              # Row-major about the x, y, z axes
+float64[9] velocity_covariance              # Row-major about the x, y, z axes
+float64[9] acceleration_covariance          # Row-major about the x, y, z axes
+float64[9] size_covariance                  # Row-major about the x, y, z axes
```

Comment:
Very minor detail but I just noticed the github warning about a missing newline at the end of the file. Maybe not relevant for *.msg files?

Reply:
New line added